features/features.go

```diff
@@ -548,4 +548,11 @@ var (
 							productScope(ocpSpecific).
 							enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 							mustRegister()
+
+	FeatureGateClusterMonitoringCRD = newFeatureGate("ClusterMonitoringCRD").
```

Comment:
Why include CRD in the name? What's the actual feature that this relates to?

Reply:
What is the CRD called?

Reply:
https://github.com/openshift/api/pull/1929/files
```

apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  annotations:
    api-approved.openshift.io: https://github.com/openshift/api/pull/1929
    api.openshift.io/merged-by-featuregates: "true"
    include.release.openshift.io/ibm-cloud-managed: "true"
    include.release.openshift.io/self-managed-high-availability: "true"
  name: clustermonitoring.config.openshift.io
spec:
  group: config.openshift.io
  names:
    kind: ClusterMonitoring
    listKind: ClusterMonitoringList
    plural: clustermonitoring
    singular: clustermonitoring
```

Reply:
How would you feel about calling this `ClusterMonitoringConfig` instead? Exposing CRD feels like a leak of an implementation detail personally

Reply:
+1 to `ClusterMonitoringConfig`

Reply:
cool, thanks

Reply:
I have a question, name ClusterMonitoring CRD as ClusterMonitoringConfig inside config is not redundant?
as example console: https://github.com/openshift/api/blob/76a71dac36a08eab1b240c6c8d4e39c813b1b12b/config/v1/types_console.go#L23 is not called consoleConfig

Reply:
I didn't mean to rename the CRD, I just think the feature gate name should be `ClusterMonitoringConfig` rather than `ClusterMonitoringCRD`. The CRD is find as `ClusterMonitoring` (or even maybe `Monitoring`)